src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -886,9 +887,17 @@ private ImmutableArray<TypeParameterSymbol> LoadTypeParameters(ref DiagnosticInf
                 else
                 {
                     var ownedParams = ImmutableArray.CreateBuilder<TypeParameterSymbol>(gpHandles.Count);
+                    UseSiteInfo<AssemblySymbol> typeParamUseSiteInfo = default;
                     for (int i = 0; i < gpHandles.Count; i++)
                     {
-                        ownedParams.Add(new PETypeParameterSymbol(moduleSymbol, this, (ushort)i, gpHandles[i]));
+                        var typeParam = new PETypeParameterSymbol(moduleSymbol, this, (ushort)i, gpHandles[i]);
+                        ownedParams.Add(typeParam);
+                        MergeUseSiteInfo(ref typeParamUseSiteInfo, typeParam.GetUseSiteInfo());
```

Comment:
>MergeUseSiteInfo(ref typeParamUseSiteInfo, typeParam.GetUseSiteInfo()); [](http://example.com/codeflow?start=24&length=71)

Changes to this function are unexpected, the pass through type parameters should be moved to ```GetUseSiteInfo```, where we do the pass for parameters.